lib/cucumber/events/step_activated.rb

```diff
@@ -7,19 +7,20 @@ module Events
     # Event fired when a step is activated
     class StepActivated < Core::Event.new(:test_step, :step_match)
 
-      # The test step that was matched.
+      # The test step that was matched.
       #
       # @return [Cucumber::Core::Test::Step]
       attr_reader :test_step
 
-      # Information about the matching definition.
+      # Information about the matching definition.
       #
       # @return [Cucumber::StepMatch]
       attr_reader :step_match
 
-      # @private
+      # @private
       def initialize(test_step, step_match)
-        @test_step, @step_match = test_step, step_match
```

Comment:
If it's more than one or two things, it can get very confusing. That's probably why the style guide just recommends putting them each on their own line.